### DISCUSSION TITLE
Unit Testing Default Empty State

### DISCUSSION THREAD

**Discussion by sonicyeti** [Kong/insomnia#3431] [Ideas]
Integrate a visually appealing default empty state for uint testing leveraging Debug's empty state as a starting point. Part of the over-arcing unit testing design sweep: https://github.com/Kong/insomnia/issues/2464

**Current empty state:**
![Screen Shot 2020-08-25 at 3 07 58 PM](https://user-images.githubusercontent.com/52717970/91217004-d6dd9800-e6e4-11ea-9edb-ff300e492a37.png)

**Debug's empty state for reference:**
![Screen Shot 2020-08-25 at 3 05 40 PM](https://user-images.githubusercontent.com/52717970/91216817-8c5c1b80-e6e4-11ea-8560-82ef8c58d2cb.png)

@erickrico

**Comment by erickrico** [comment]
what are the main actions and commands you can do on test mode? @sonicyeti

**Comment by sonicyeti** [comment]
> what are the main actions and commands you can do on test mode? @sonicyeti

@erickrico From an empty state the first (and only) action would be to create a new test suite. 

@nijikokun @develohpanda Would it be awesome to almost treat it like a guided scenario where we prompt them to create a new test suite from scratch or start with a template that we drop in? I'd love that, especially given the UX pattern here where you really need to follow a very linear path to get going.

**Comment by develohpanda** [comment]
Just some clear buttons in the big open area that say something like "Create a new test suite", and then "create a new test" once an empty test suite is selected. Can be guided, but IMO needs to be as few clicks as possible.
